src/lbaf/Utils/lbsPath.py

```diff
@@ -10,7 +10,7 @@ def abspath(path: str, relative_to: Optional[str] = None) -> Optional[str]:
 
     :param path: the input path
     :param relative_to: the base path, defaults to None (None = the current working directory)
-    :return: an absolute path
+    :returns: an absolute path
```

Comment:
not a good name for a path

Reply:
@ppebay what do you suggest ?

Reply:
`absolute_path`?

Reply:
@ppebay Not sure it is logic to rename. In fact this function accepts both relative or absolute path but returns an an absolute path